dev/custom-pages/internet-privacy.html

```diff
@@ -0,0 +1,39 @@
+<div class="container">
```

Comment:
Remove this .container--it's not needed and it causes some nesting issues throughout the page.